playbooks/deploy_cluster.yml

```diff
@@ -1,44 +1,9 @@
 ---
 - import_playbook: init/main.yml
 
-- import_playbook: openshift-checks/private/install.yml
+- import_playbook: control_plane.yml
```

Comment:
Are users going to encouraged to call control_plane.yml, or is this just for dev/test purposes?

As-is, init/main.yml will be called 3 times; we don't want that, we need to come up with a way that init/main.yml is only called once per run of ansible-playbook.

I suggest we don't do this at all.

Reply:
I think eventually control_plane is the only part that will be in openshift-ansible, and the other parts will be called by different tools (or at a higher level).

Reply:
I agree the init part, and this being available to end users, are both issues.  Do we have a practical way to run-once an included playbook?  If we don't, then I think we should get rid of deploy_cluster and have control_plane and components (or whatever we call it) and do them in sequence.  Installing a control plane has nothing to do with the components on top of it.

Reply:
I would prefer to have the split of playbooks entirely as you suggest.  However, I don't think we want to do that this late in the cycle.

Reply:
As a way forward, I think we can simply make the two parts "private" in a separate folder and include them here, and in AWS and GCP, with comments about pre and post conditions.

Reply:
Updated with the less contentious path, including a thorough description of what the two playbooks are for.